module1/homework/lengthSort/sort.cpp

```diff
@@ -0,0 +1,12 @@
+#include "sort.hpp"
+
+
+std::deque<std::string> lengthSort(std::forward_list<std::string>& strings) {
+    std::deque<std::string> sorted_strings{};
+
+    strings.sort();
```

Comment:
this will not work. Try to add more cases in test. Like:
```
    {"q"},
    {"fl"},
    {"dla"},
    {"clam"},
    {"blama"},
    {"alamak"},
```
And test will fail.

Reply:
@nauka-programowania-MA We have weak test cases ;)